joomla/joomla-cms
# [com_content] Filter on publish_up and publish_down

### Summary of Changes

This PR adds the possibility to filter article on the "Publish Up" and "Finish Publishing" date.
### Testing Instructions
1. Apply this PR
2. Create some article with different "publish up" and "finish publishing" dates
3. Select the following option in the "Sort table by" field and confirm that the articles are sorted correctly:
   - Start Publishing ascending
   - Start Publishing descending
   - Finish Publishing ascending
   - Finish Publishing descending
